igvc_gazebo/launch/sim_detector.launch

```diff
@@ -0,0 +1,16 @@
+<?xml version="1.0"?>
+
+<!-- roslaunch igvc_gazebo qualification.launch -->
```

Comment:
I'd remove these, if you want to create documentation I would put them there, you will get more used to the command later on.